sql/psi_memory_key.cc

```diff
@@ -411,12 +411,12 @@ static PSI_memory_info all_server_memory[] = {
      "Mem root for temporary objects allocated while dropping tables or the "
      "whole database."},
     {&key_memory_rm_db_mdl_reqs_root, "rm_db::mdl_reqs_root", PSI_FLAG_THREAD,
+     0, "Mem root for allocating MDL requests while dropping datbase."}
```

Comment:
Typo: datbase

Reply:
Looks a typo in upstream as well. https://github.com/mysql/mysql-server/blob/mysql-8.0.27/sql/psi_memory_key.cc#L371